addons/website/models/res_config_settings.py

```diff
@@ -133,7 +133,7 @@ def _inverse_shared_user_account(self):
     @api.depends('website_id.auth_signup_uninvited')
     def _compute_auth_signup_uninvited(self):
         for config in self:
-            config.auth_signup_uninvited = config.website_id.auth_signup_uninvited
+            config.auth_signup_uninvited = config.website_id.auth_signup_uninvited or 'b2b'
```

Comment:
This is required for existing 15.3 db that did not update their views and do not have eCommerce installed.

Reply:
Good catch.
Could you write that in a comment in the code, mentioning it will be `reverted in master (saas-15.4)`

Reply:
Sure!